Xamarin.Forms.ControlGallery.WindowsUniversal/Xamarin.Forms.ControlGallery.WindowsUniversal.csproj

```diff
@@ -61,6 +61,7 @@
     <ErrorReport>prompt</ErrorReport>
     <Prefer32Bit>true</Prefer32Bit>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <PostBuildEvent>if exist "$(TargetDir)AppX\" (xcopy "$(TargetDir)*.pdb" "$(TargetDir)AppX\" /Y)</PostBuildEvent>
```

Comment:
Yea it's annoying UWP debugging needs anything :-/ 

We fixed this on the control gallery project here
https://github.com/xamarin/Xamarin.Forms/commit/069f458f7de24d868c1874111af1f835b3d30f45

I'm wondering if a similar addition onto Xamarin.Forms.Core  would be a good way to go.
```xml	
<PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
		<DebugType>full</DebugType>
	</PropertyGroup>
```

I tried and it looks to fix the debugging issue on the core

Reply:
@PureWeen can you make a pr so we make sure it passes CI, then we should test the package to make sure nothing change.. (table to debug just with nuget pack)

Reply:
https://github.com/xamarin/Xamarin.Forms/pull/3040